CalculatorTests/Pages/CalculatorPage.cs

```diff
@@ -0,0 +1,158 @@
+﻿using OpenQA.Selenium;
+using OpenQA.Selenium.Support.UI;
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CalculatorTests.Pages
+{
+    public class CalculatorPage
+    {
+        private IWebDriver _driver;
+        private string value;
+
+        public CalculatorPage(IWebDriver driver)
+        {
+            _driver = driver;
+        }
+
+        public IWebElement DepAmountFld
+        {
+            get
+            {
+                return _driver.FindElement(By.Id("amount"));
+            }
+        }
+
+        public IWebElement RateInterestFld
+        {
+            get
+            {
+                return _driver.FindElement(By.Id("percent"));
+            }
+        }
+
+        public IWebElement InvestTermFld
+        {
+            get
+            {
+                return _driver.FindElement(By.Id("term"));
+            }
+        }
+
+        public SelectElement DateDayDrdwn
+        {
+            get
+            {
+                return new SelectElement(_driver.FindElement(By.Id("day")));
+            }
+        }
+        public SelectElement DateMonthDrdwn
```

Comment:
Please add blank line between end of one item and begin of the next one.
```suggestion
        }
        
        public SelectElement DateMonthDrdwn
```